src/main/resources/anniv5Resources/localization/eng/gemspack/UIstrings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "${ModID}:GemsPack": {
+    "TEXT": [
+      "Jeweler's Pack",
+      "Pure Support Pack. Contains a selection of 0-cost Gems that can be placed into cards at Rest site.",
+      "MichaelMayhem"
+    ]
+  },
+  "${ModID}:GemsPackHat": {
+    "TEXT": [
+      "It's not gaudy at all!",
+      "Crowned Jewel"
+    ]
+  },
+  "${ModID}:EnhanceBonfireOptions": {
+    "TEXT": [
+      "Enhance",
+      "(Free Action) Spend 20 Gold. Place a Gem into a card.",
```

Comment:
Just a note, you could've not hardcoded the cost here, that way changes to socketing price would be easy to make

Reply:
While true for the UIString, this can't be done for the Keyword string right?